<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wqDock</class>
 <widget class="QDockWidget" name="wqDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WQ Graph</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Knoten Nummer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="wqKnoten"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addData">
         <property name="text">
          <string>Pegel Daten Hinzuf√ºgen</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTableView" name="wqTable"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="PlotWidget" name="wqPlot"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
